app/models/host_status/build_status.rb

```diff
@@ -5,21 +5,28 @@ class BuildStatus < Status
     BUILD_FAILED = 3
     BUILT = 0
 
+    @@build_status_help_text = {}
+
     def self.status_name
       N_("Build")
     end
 
     def to_label(options = {})
       case to_status
         when PENDING
+          @@build_status_help_text["Pending installation"] = "Wait for sometime and if build fails, rebuild again"
```

Comment:
How about adding a method to `to_label_help` that would have the same case, but instead of label would return the full help. Then we wouldn't need @@ var nor the helper method. In the view we'd check if this methods returns nil or string, in case of string, we'd use it as help.

I guess that means changing the current host page a bit more, but I believe, that would be cleaner. Let me know, if you need help or I should explain more.